src/senaite/core/browser/worksheet/configure.zcml

```diff
@@ -0,0 +1,136 @@
+<configure
+        xmlns="http://namespaces.zope.org/zope"
+        xmlns:plone="http://namespaces.plone.org/plone"
+        xmlns:browser="http://namespaces.zope.org/browser"
+        i18n_domain="senaite.core">
+
+  <browser:page
+      for="senaite.core.interfaces.IWorksheets"
+      name="view"
+      class=".views.WorksheetsView"
+      permission="zope.Public"
```

Comment:
This should be `permission="cmf.ListFolderContents"` so only users able to list the contents of `worksheets` folder have access granted to this view.